lib/queue_classic/worker.rb

```diff
@@ -112,7 +114,10 @@ def process(queue, job)
           queue.delete(job[:id])
           finished = true
         end
-      rescue => e
+      rescue StandardError, ScriptError, NoMemoryError => e
```

Comment:
This is the only other possible surprise I can see, if there were some other error we were silently handling here up til now.

Reply:
yep, but, prolly we shouldn't be right? but might cause a few new unseen errors?

Reply:
Right, I don't expect there to be any, and this isn't a blocker to upgrading... just something to be aware of I guess? If we see new errors in Sentry, they're probably from here?